README.md

```diff
@@ -61,57 +63,55 @@ Service to the `c.JupyterHub.services` list:
 ```python
 c.JupyterHub.services = [
     {
-        'name': 'idle-culler',
-        # 'admin': True,
-        'command': [
+        "name": "jupyterhub-idle-culler",
+        "command": [
             sys.executable,
-            '-m', 'jupyterhub_idle_culler',
-            '--timeout=3600'
+            "-m", "jupyterhub_idle_culler",
+            "--timeout=3600",
         ],
+        # "admin": True,
     }
 ]
 ```
 
 where:
 
-- `'admin': True` indicates that the Service requires admin permissions so
-  it can shut down arbitrary user notebooks
-  (only for jupyterhub < 2.0; see [above][permissions]), and
-- `'command'` indicates that the Service will be managed by the Hub.
+- `"command"` indicates that the Service will be managed by the Hub, and
+- `"admin": True` grants admin permissions to this Service and is only meant for
+  use with jupyterhub < 2.0; see [above][permissions].
```

Comment:
I moved the admin: True part to the end as it was less important than the command that is relevant to jupyterhub 1 and 2.